### ISSUE TITLE
Feature Suggestion/Request: Mouse-less plotting

### ISSUE THREAD

**Issue by gchapyzh** [endless-sky/endless-sky#945]
This is a simple one: When looking at the map, one should be able to cycle through jumps with J (already implemented) and select the jump with Enter, allowing them the cycle through the jumps from the new target. Del/Backspace should delete the previous jump plotted.

So, if you want to go to a system 2 jumps way, you would select the jump by cycling through all the available ones, Enter, select the next jump, Enter, and exit the map with a ready plot.

**Comment by endless-sky** [comment]
Added in a0fa934. At the same time, I also changed how the jump destinations cycle so that they always go in order by angle, starting at 12:00 and going in a clockwise direction.

To match how click and shift-click work, what I did was this:

* If you press the jump key, it cycles through options for the last jump on your path.
* If you press Shift + jump, it adds another jump to your path.
* If you press delete, it removes one jump from the end of your path.

So, to select a two-jump path, you'd press jump to cycle through to the first system you want to go to, then press Shift + jump, and then press jump until the second system you want to go to is selected.
